workshop1/slides/src/fonts/local.tex

```diff
@@ -0,0 +1,19 @@
+\setromanfont{Norwester}[
+  Path           = fonts/ ,
```

Comment:
What's the license for the fonts? Can we add them to the repository?

Reply:
I'll make a list

Reply:
Norwester: SIL Open Font License v1.10 (https://jamiewilson.io/norwester/)
Montserrat: SIL Open Font License v1.10 (https://github.com/JulietaUla/Montserrat/blob/master/OFL.txt)
Fantasque Sans Mono: SIL Open Font License v1.10 (https://github.com/belluzj/fantasque-sans/blob/master/LICENSE.txt)

Redistributing SIL OFL fonts requires to retain a link to the license:

> Original or Modified Versions of the Font Software may be bundled,
> redistributed and/or sold with any software, provided that each copy
> contains the above copyright notice and this license

Reply:
Added licenses

Reply:
Added the licenses